### PR TITLE
added differential analysis

### PR DESCRIPTION
**Pull request by angelinetsui**

### CODE REVIEW

File: pilot_data_analysis.Rmd [hunk 1]
```diff
@@ -219,76 +220,65 @@ ggplot(ms, aes(x = t, y = looking, col = region)) +
...
-                                       subid_trial == "MB2-124" ~ 1,
-                                       subid_trial == "MB2-126" ~ 1,
-                                       subid_trial == "mb2p067" ~ 1,
-                                       subid_trial == "mb2p103" ~ 1,
-                                       subid_trial == "mb2p114" ~ NA_real_,
-                                       subid_trial == "mb2p126" ~ 0,
-                                       subid_trial == "mb2p127" ~ 1,
-                                       TRUE ~ aoi_dummy)) %>% 
-  filter(!is.na(first_look_location))
+         subid_trial = paste0(subid, sep = "_trial_", trial_num)) %>%
```
Comment (mahowak) [inline pilot_data_analysis.Rmd:232]: did you want to comment all this out?

File: pilot_data_analysis.Rmd [hunk 2]
```diff
@@ -219,76 +220,65 @@ ggplot(ms, aes(x = t, y = looking, col = region)) +
...
+#                                        subid_trial == "mb2p126" ~ 0,
+#                                        subid_trial == "mb2p127" ~ 1,
+#                                        TRUE ~ aoi_dummy)) %>% 
+#   filter(!is.na(first_look_location))
 ```
 
 ## Data visualization across different test trials
 
-```{r}
+```{r, eval = FALSE}
```
Comment (mahowak) [inline pilot_data_analysis.Rmd:281]: did you mean this eval?

File: pilot_data_analysis.Rmd [hunk 3]
```diff
@@ -309,31 +299,32 @@ df_t_first_aoi %>%
...
 ## Data visualization across different test trials
 ```{r}
 ms_proportion_trials <- d %>%
-  group_by(t, trial_num) %>%
+  group_by(t, trial_num, experiment_num) %>%
   summarise(target = mean(aoi == "target", na.rm = TRUE),
             distractor = mean(aoi == "distractor", na.rm = TRUE)) %>%
   mutate(proportion_to_target = target / (target + distractor)) %>% 
-  filter(t >= -4000 & t <= 120+ 4000)
+  filter(t >= -500 & t <= 120+ 500)
```
Comment (mahowak) [inline pilot_data_analysis.Rmd:327]: Before, we had been using 4s before the pod, which was kind of long. But maybe 500ms is short?

File: pilot_data_analysis.Rmd (hunk 1 above)
Comment (angelinetsui) [reply pilot_data_analysis.Rmd:232]: I will cut this out after Gal and I found a new way to compute the first look

File: pilot_data_analysis.Rmd (hunk 2 above)
Comment (angelinetsui) [reply pilot_data_analysis.Rmd:281]: ahhh, yes

File: pilot_data_analysis.Rmd (hunk 3 above)
Comment (angelinetsui) [reply pilot_data_analysis.Rmd:327]: Yes, 4sec is too long, but I don't really have a better idea of how long we should expect, I can extend it to 1000ms, but I think 500ms is still good in the sense that even we focus on such a short period of time, the overall evidence suggested that kids looked at target more than distractor pre- and post-POD

@mcfrank , what do you think? Should we set 1000ms or 500ms for the anticipatory window?

**Comment by mcfrank** [comment]
Sorry, I don't have a lot of context here but I think 500ms is very short.
I would imagine that 1000 or even 2000 ms would be more reasonable...

Mike

**Comment by mahowak** [comment]
Yeah I agree. The original thinking in pilot 1a was, I think, to have the
anticipation window be the period of time after Bear went in the tunnel but
before he emerged. This was why we were using around 4000ms. Bear is fully
gone for around 3860ms in this version.

I think in Version A, there were about 2000ms between when Bear popped up
in the window and when he emerged.

**Comment by angelinetsui** [comment]
Thanks all, I have changed it to 2000 ms and submit pull request too

**Pull request merged**